sfputil/main.py

```diff
@@ -903,6 +962,258 @@ def reset(port_name):
 
         i += 1
 
+# 'firmware' subgroup
+@cli.group()
+def firmware():
+    """Download/Upgrade firmware on the transceiver"""
+    pass
+
+def run_firmware(port_name, mode):
```

Comment:
>run_firmware [](http://example.com/codeflow?start=4&length=12)

Add some comment to explain the return value

Reply:
Is it possible that you return non 1 and not exiting?

Reply:
i don't see. the caller of run_firmware() will exit if 'run_firmare' return non 1